README.md

```diff
@@ -1123,39 +1123,39 @@ Job.aasm.states.map(&:name)
 job = Job.new
 
 # show all permitted states (from initial state)
-job.aasm.states(:permitted => true).map(&:name)
+job.aasm.states(permitted: true).map(&:name)
 #=> [:running]
 
 job.run
-job.aasm.states(:permitted => true).map(&:name)
+job.aasm.states(permitted: true).map(&:name)
 #=> [:sleeping]
 
 # show all non permitted states
-job.aasm.states(:permitted => false).map(&:name)
+job.aasm.states(permitted: false).map(&:name)
 #=> [:cleaning]
 
 # show all possible (triggerable) events from the current state
 job.aasm.events.map(&:name)
 #=> [:clean, :sleep]
 
 # show all permitted events
-job.aasm.events(:permitted => true).map(&:name)
+job.aasm.events(permitted: true).map(&:name)
 #=> [:sleep]
 
 # show all non permitted events
-job.aasm.events(:permitted => false).map(&:name)
+job.aasm.events(permitted: false).map(&:name)
 #=> [:clean]
 
 # show all possible events except a specific one
-job.aasm.events(:reject => :sleep).map(&:name)
+job.aasm.events(reject: :sleep).map(&:name)
 #=> [:clean]
 
 # list states for select
 Job.aasm.states_for_select
-=> [["Sleeping", "sleeping"], ["Running", "running"], ["Cleaning", "cleaning"]]
+#=> [["Sleeping", "sleeping"], ["Running", "running"], ["Cleaning", "cleaning"]]
```

Comment:
why commented this line ?

Reply:
I was hoping to bring it more in line with the prior examples, which used a comment to denote it was a returned value. This one wasn't consistent with that.

Reply:
ok 👍